src/components/about/OrgBookData.vue

```diff
@@ -57,8 +58,8 @@ import { mapGetters } from "vuex";
 })
 export default class OrgBookData extends Vue {
   formattedDescription(type: ICredentialType): string {
-    return (
-      type.schema_label?.translations?.[i18n.locale]?.description ||
+    return ( // change to type.schema_label?.[i18n.locale]?.description after update to backend
```

Comment:
Please place comment on its own line and prefix them with either `TODO:` or `FIXME:`. The prefix makes it easier to search through places that need to be worked on in the future. Same with remaining comments.